src/modules/createLike.js

```diff
@@ -0,0 +1,19 @@
+import showLikes from './showLikes.js';
+import retrieveData from './retrieveData.js';
+
+const createLike = async (api, itemId) => {
+  await fetch(api, {
+    method: 'POST',
+    body: JSON.stringify({ item_id: `${itemId}` }),
+    headers: {
+      'Content-Type': 'application/json; charset=utf-8',
+    },
+  }).then(() => retrieveData(api)
+    .then((response) => {
+      showLikes(response);
+    })).catch((error) => {
+    throw error;
+  });
+};
```

Comment:
**Changes Required**
- [ ] Fair effort on using the `async await`  promise syntactic sugar. However, you're mixing two syntax here, and the `async` keyword in this context is redundant. May we please remove the `.then` and stick to the `async await` format.  You could try your code in a `trycatch` block.

Reply:
Changed ✔